frontends/concrete-python/tests/compilation/test_decorators.py

```diff
@@ -268,7 +268,7 @@ def circuit1(x: int):
             return x + 42
 
     assert str(excinfo.value) == (
-        f"Annotation {str(int)} for argument 'x' is not valid "
+        f"Annotation {int!s} for argument 'x' is not valid "
```

Comment:
This syntax is a lot less understandable IMO, do we really want to have this?

Also, is it supported on 3.8?

Reply:
I can certainly revert and ignore the corresponding error if the other reviewers want it

Reply:
Wdyt guys @BourgerieQuentin @rudy-6-4 ?

Reply:
And what about 3.8 support?

Reply:
Not strong opinion.. But as well that should be supported in 3.8.

Reply:
I've removed those things